lib/challenge.js

```diff
@@ -1,5 +1,33 @@
 'use strict';
 
+let milli = 1000000;
+let sumTen = 0;
+let sumTwo = 0;
+
+String.prototype.reverse = function() {
+	return this.split('').reverse().join('');
+};
+
+let baseTen = function() {
+  for (let i =0; i < milli; i++) {
+    let iToString = i.toString();
+    if ( iToString === iToString.reverse() && (i % 10) === i) {
```

Comment:
Why are you testing for `(i % 10) === i`? This will only be true for numbers between 0 and 9, inclusive.